### PR TITLE
Added function

### PR DESCRIPTION
**Pull request by cst0**
hfcutfrcxutfc

### CODE REVIEW

**Comment by Taylordevinn** [comment]
no

File: ourlib.c [hunk 1]
```diff
@@ -59,3 +59,7 @@ double _div(int x, int y) {
 int _abs(int arg) {
     return 0;
 }
+
+int fakefunction(void) {
+    return 0;
```
Comment (andechen) [inline ourlib.c:64]: this does nothing

**Review by andechen — APPROVED** [review]
wlook good

**Comment by andechen** [comment]
is that...an M5 stainless steel bolt?!?!

**Pull request merged**